src/main/java/com/github/jmchilton/blend4j/galaxy/WorkflowsClient.java

```diff
@@ -22,4 +22,11 @@
   ClientResponse runWorkflowResponse(WorkflowInputs workflowInputs);
 
   WorkflowOutputs runWorkflow(WorkflowInputs workflowInputs);
+  
+  /**
+   * Deletes the workflow with the given id.  This action is irreversible.
+   * @param id The id of the workflow to delete.
+   * @return A {@link ClientResponse} for this request.
+   */
+  ClientResponse deleteWorkflowResponse(String id);
```

Comment:
I dunno, I guess it's hard to justify spending time writing it if we aren't going to use it...

If it doesn't take much time, then go for it.